File: backend/micro_epoweredmove_reviews_reservations/src/main/java/com/kosdiam/epoweredmove/models/Review.java
package com.kosdiam.epoweredmove.models;

import java.io.Serializable;

import com.kosdiam.epoweredmove.models.enums.ReviewStatus;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Review implements Serializable {
    private String id;
    private String comments;
    private Integer rating;
    private ReviewStatus status;
    private String userId;
    private String chargingStationId;
    private Long timestamp;
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/Review.kt
package com.kosdiam.epoweredmove.models

import android.os.Parcelable
import com.kosdiam.epoweredmove.models.enums.ReviewStatus
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Review(
    val id: String,
    var comments: String?,
    var rating: Int,
    val status: ReviewStatus,
    var timestamp: Long,
    var userId: String,
    val userObj: User,
    var chargingStationId: String,
    val chargingStationObj: ChargingStation
) : Parcelable{
    constructor(): this("", "", 0, ReviewStatus.ACTIVE, 0, "", User(), "", ChargingStation())
}
